src/bot.py

```diff
@@ -15,7 +15,9 @@
 print("""
                        TinyFlowerBeds
 
-            Copyright (C) 2018-2019 AlwaysLivid
+Copyright (C) 2018-2020 AlwaysLivid, Shankhanil, karl01101101

```

Comment:
Could you make the text align towards the center? Additionally, could you change the text to `TinyFlowerBeds contributors` and then point people to this repository instead?

The text is a bit unreasonably long.

Reply:
Sure